scenes/game/local_game.gd

```diff
@@ -27,9 +27,12 @@ var all_cards : Array = []
 var game_over : bool = false
 var game_over_winning_player : Player = null
 var active_strike : Strike = null
+var active_character_action : bool = false
 var active_exceed : bool = false
 var active_overdrive : bool = false
+var active_revert : bool = false
```

Comment:
I think instead of having the revert action be a completely separate thing, we should have it be another character_action.
We'd have to make the definition support listing multiple character actions, but I think overall this will be cleaner on a couple fronts:
1) in game.gd, no special case logic for s6_revert, just support looking for an array of character actions and making buttons for them
2) in localgame/remotegame, adding a parameter to submit/do_character_action for which character action you're doing
3) In the definition file, the 2nd character action is just move a card from hand to gauge, revert, and then take another action.

This may also make things simpler for adding the AI features for this.
Also, the code you've added for active_character_action will be the same as what you needed for active_revert, so some reuse there too.

Reply:
As for adding the array, it should be pretty easy to have get_character_action/can_do_character_action take as input an int, and then the code in those functions can check if it is an array or a single character action and do the appropriate thing.
I mention this just so we don't have to go update every character definition to be an array, though that actually wouldn't be that bad either so maybe just do that for code simplicity?

Reply:
Will work on this once I clean up some of the smaller stuff. What I think I may try to do is add support for multiple character actions, but keep in the "s6_revert_action" for the json file, then have a definition for the revert action that automatically gets added to the list of exceed actions if that's set to true (just to avoid needing to copy it for every s6 character)

Reply:
I think I would prefer to have it just all in the character json for clarity/simplicity rather than having to look somewhere else to see what s6_revert_action does.
Then there's less checking of random fields on the character json in the code.
Copying it to each character shouldn't be an issue since when making a new s6 you'll just copy an existing s6 and replace stuff.

The goal is to not have to have like can_do_s6_revert and do_s6_revert in gamewrapper/localgame/remotegame at all.  And in game.gd it just iterates through all character actions and makes buttons for them.